v1.0/docs/getting-started-guides/fedora/fedora_ansible_config.md

```diff
@@ -1,5 +1,6 @@
 ---
 layout: docwithnav
+title: Prerequisites
```

Comment:
Ugh, we have a bunch of docs that have "titles" in the form of dash-underlines, which renders as H2.

@RichieEscarez ! help! We have no consistency.  We need a doc style guide.

I would say you take the first match of: "^# .*", an equal-underlined line, a dash-underlined line, the filename minus docsdir prefix.